includes/class-amp-story-post-type.php

```diff
@@ -1048,23 +1080,32 @@ public static function render_block_latest_stories( $attributes ) {
 			'meta_key'         => '_thumbnail_id',
 		);
 		$story_query     = new WP_Query( $args );
-		$min_height      = self::get_featured_image_minimum_height( $story_query->posts );
 		$class           = 'amp-block-latest-stories';
 		if ( isset( $attributes['className'] ) ) {
 			$class .= ' ' . $attributes['className'];
 		}
+		$size       = self::STORY_CARD_IMAGE_SIZE;
```

Comment:
This changes the `<amp-carousel>` image size to `626 x 929`. This portrait size is a little strange for the carousel.

But the `<amp-carousel>` [only allows layouts](https://www.ampproject.org/docs/reference/components/amp-carousel) of `fixed` and `fixed-height`. That means this has to add a `height` to the `<amp-carousel>`

It's hard to calculate the actual display height for the `928 x 696` landscape image:
<img width="1433" alt="image-example" src="https://user-images.githubusercontent.com/4063887/55694407-b428d300-5979-11e9-8d90-24b256614732.png">


For landscape images, it's more likely that the container won't have room for the entire width, which could reduce the height.

For example, the `amp-story-poster-landscape` image size is `928 x 696`. But the global `$content_width` can sometimes be `620 px`. Or the block could be in a column, or something with an unpredictable width.